kogito-timer-quarkus/README.md

```diff
@@ -15,11 +15,66 @@ This example shows
 * optionally use Job Service that allows to externalize time tracking to separate service and by that offload the runtime service
 	
 	
-Intermediate timer event
-<p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+* Intermediate timer event
+<p align="center"><img width=75% height=50% src="docs/images/timers.png"></p>
+
+* Intermediate timer Diagram Properties
+<p align="center"><img src="docs/images/timersDiagramProperties.png"></p>
+
+* Intermediate timer Diagram Properties
+<p align="center"><img src="docs/images/timersDiagramProperties2.png"></p>
+
+* Intermediate timer Before Timer
+<p align="center"><img src="docs/images/timersBeforeTimerScriptTask.png"></p>
+
+* Intermediate timer Timer
+<p align="center"><img src="docs/images/timersTimer.png"></p>
+
+* Intermediate timer After Timer 
+<p align="center"><img src="docs/images/timersAfterTimerScriptCall.png"></p>
+
+* Boundary timer event
+<p align="center"><img width=75% height=50% src="docs/images/timerOnTask.png"></p>
+
+* Boundary timer Diagram Properties 
+<p align="center"><img src="docs/images/timerOnTaskDiagramProperties.png"></p>
+
+* Boundary timer Diagram Properties 
+<p align="center"><img src="docs/images/timerOnTaskDiagramProperties2.png"></p>
+
+* Boundary timer Before Timer 
+<p align="center"><img src="docs/images/timerOnTaskBeforeTimerScriptCall.png"></p>
+
+* Boundary timer User Task
+<p align="center"><img src="docs/images/timerOnTaskUserTask1.png"></p>
+
+* Boundary timer User Task
+<p align="center"><img src="docs/images/timerOnTaskUserTask2.png"></p>
+
+* Boundary timer Timer
+<p align="center"><img src="docs/images/timerOnTaskBoundaryTimer.png"></p>
+
+* Boundary timer After Timer
+<p align="center"><img src="docs/images/timerOnTaskAfterTimerScriptCall.png"></p>
+
+* Cycle timer event
+<p align="center"><img width=75% height=50% src="docs/images/timersCycleProcess.png"></p>
+
+* Cycle timer Diagram Properties
+<p align="center"><img src="docs/images/timersCycleDiagramProperties.png"></p>
```

Comment:
maybe change the name property in screenshot/bpmn file to `timerscycle`

Reply:
fixed @radtriste